extension/package.json

```diff
@@ -5,14 +5,13 @@
   "license": "MIT",
   "type": "module",
   "scripts": {
-    "check-coverage": "nyc check-coverage --lines 89",
-    "test": "npm run check && npm run cover:unit && npm run cover:integration && npm run cover:e2e && npm run check-coverage",
-    "cover:unit": "nyc --silent npm run unit",
-    "cover:integration": "nyc --silent --no-clean npm run integration",
-    "cover:e2e": "nyc --silent --no-clean npm run e2e",
-    "unit": "nyc mocha --file ./test/unit/mock-default-confg.js \"test/unit/**/*.spec.js\"",
-    "integration": "nyc mocha --file ./test/initial-integration-test-set-up.js \"test/integration/**/*.spec.js\"",
-    "e2e": "nyc mocha --file ./test/initial-integration-test-set-up.js \"test/e2e/**/*.spec.js\"",
+    "test-coverage-run": "c8 npm run test-run && npm run check-coverage",
+    "test-run": "npm run unit && npm run integration && npm run e2e",
+    "check-coverage": "c8 check-coverage --lines 89",
+    "test": "npm run check && npm run test-coverage-run",
```

Comment:
Now you can remove `test-coverage-run` and put its content to `test` and things will be back as they were before.

Reply:
No, It looks good this way. 
test -> which includes check and other test runs with coverage.
test-coverage-run -> without lint check.

Reply:
I don't see a case where you run all the tests and don't want lint check. Even locally it doesn't make sense - when you run all tests, you want to check if it will be ok in CI, and in CI you always have lint check before. So I recommend to remove a command without lint check.

Reply:
In CI it is better to have lint check and tests separately. So we have `check` and `test-coverage-run`.
`test` is for local development.

Reply:
I don't see what is the benefit of having 2 commands if you never use them separately. WDYT?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/976/commits/2057cb475054f2c07c04dd50eae390c28e6e70a4